exercises/queen-attack/src/Queens.hs

```diff
@@ -0,0 +1,7 @@
+module Queens (boardString, canAttack) where
+
+boardString :: Maybe (Int, Int) -> Maybe (Int, Int) -> String
+boardString = undefined
```

Comment:
it may not be immediately obvious that the first is white and the second is black, but maybe that's OK it's easily discoverable from reading the tests. unless the goal is to make it obvious without having to read the tests